etc/kernel-launchers/R/scripts/launch_IRkernel.R

```diff
@@ -1,7 +1,7 @@
-library(SparkR)
 library(argparser)
 library(jsonlite)
 
+require("SparkR")
```

Comment:
Is this going to be ok with the vanilla kernel?

Reply:
Yes. See last paragraph of PR comment. Use of `library` is more binding than `require`.  Seems like `require` Is poorly named IMHO.